src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -806,6 +807,36 @@ namespace
             object_assembly_instance->transform_sequence()
                 .set_transform(0.0, transform);
 
+            const int ObjectMotionBlur = 1;
+            if (node->GetMotBlurOnOff(time) && node->MotBlur() == ObjectMotionBlur)
+            {
+                Control* tm_controller = node->GetTMController();
+
+                const bool is_animated =
+                    tm_controller->GetPositionController()->IsAnimated() > 0 ||
+                    tm_controller->GetRotationController()->IsAnimated() > 0 ||
+                    tm_controller->GetScaleController()->IsAnimated() > 0;
+
+                if (is_animated)
+                {
+                    float shutter_duration = 0.5f;
+                    if (view_node != nullptr)
+                    {
```

Comment:
The correct thing to do might be to remove everything related to the camera here, and simply set the object's transform at time = 0 (frame 0) and time = 1 (frame 1).

That means you would need to call `GetObjTMAfterWSM()` at `time` and `time + GetTicksPerFrame()`.